packages/strapi-design-system/src/ToggleInput/ToggleInput.tsx

```diff
@@ -19,21 +18,36 @@ const ClearButton = styled(TextButton)`
   margin-left: auto;
 `;
 
+interface ToggleInputProps {
```

Comment:
This should probably extend `ToggleCheckboxProps` (if it exists yet) because we pass the spread to `ToggleCheckbox`. Also I think you should also extend some of the `FieldProps` which means you don't need to retype `disabled` or `error` etc as they're passed to that component :)